docs/reference/commandline/node_pause.md

```diff
@@ -0,0 +1,27 @@
+<!--[metadata]>
++++
+title = "node pause"
+description = "The node pause command description and usage"
+keywords = ["node, pause"]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# node pause
+
+    Usage:  docker node pause NODE [NODE...]
+
+    Pause a node in the swarm
+
+Pause a node into the swarm. This command targets a docker engine that is a manager in the swarm cluster.
```

Comment:
Explain what pausing means: no new tasks will be assigned to this node until it is reactivated with "docker node activate".

Same comment as above about "This command targets a docker engine...".

s/into/in/